From 6e3e8ffb5f478be2b9f8da4480171ba7ce02a1f0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=89=8B=E7=93=9C=E4=B8=80=E5=8D=81=E9=9B=AA?=
 <nanaeonn@outlook.com>
Date: Thu, 23 Oct 2025 13:28:52 +0800
Subject: [PATCH 4/6] Update SILK decoder include paths

Changed the include paths for SILK v3 decoder headers to reference the new location at '../../silk-v3-decoder/silk/interface/'. This aligns the code with the updated directory structure.
---
 libavcodec/silkdec.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libavcodec/silkdec.c b/libavcodec/silkdec.c
index d66d611b6b..f3b45cf8da 100644
--- a/libavcodec/silkdec.c
+++ b/libavcodec/silkdec.c
@@ -7,10 +7,10 @@
 #include "libavutil/channel_layout.h"
 #include "libavutil/opt.h"
 #include "libavutil/mem.h"
-#include "libavcodec/silk_v3_decoder/interface/SKP_Silk_SDK_API.h"
-#include "libavcodec/silk_v3_decoder/interface/SKP_Silk_control.h"
-#include "libavcodec/silk_v3_decoder/interface/SKP_Silk_typedef.h"
-#include "libavcodec/silk_v3_decoder/interface/SKP_Silk_errors.h"
+#include "../../silk-v3-decoder/silk/interface/SKP_Silk_SDK_API.h"
+#include "../../silk-v3-decoder/silk/interface/SKP_Silk_control.h"
+#include "../../silk-v3-decoder/silk/interface/SKP_Silk_typedef.h"
+#include "../../silk-v3-decoder/silk/interface/SKP_Silk_errors.h"
 
 #define SILK_SAMPLE_RATE 24000
 #define MAX_BYTES_PER_FRAME 250
-- 
2.47.0.windows.2

